app/services/partners/request_create_service.rb

```diff
@@ -54,16 +54,33 @@ def populate_item_request(partner_request)
         attrs['item_id'].blank? && attrs['quantity'].blank?
       end
 
-      item_requests = formatted_line_items.map do |ira|
-        Partners::ItemRequest.new(
-          item_id: ira['item_id'],
-          quantity: ira['quantity'],
-          children: ira['children'] || [], # will create ChildItemRequests if there are any
-          name: fetch_organization_item_name(ira['item_id']),
-          partner_key: fetch_organization_partner_key(ira['item_id'])
-        )
+      items = {}
+
+      formatted_line_items.each do |input_item|
+        pre_existing_entry = items[input_item['item_id']]
+        if pre_existing_entry
+          pre_existing_entry.quantity = (pre_existing_entry.quantity.to_i + input_item['quantity'].to_i).to_s
```

Comment:
`.to_s` meh, but I get it ... the actual database for Partner::ItemRequest quantity is a string! hmmmmm

Reply:
soooo annoying that the schema uses string for quantity